Library/RemoteConfig/RemoteConfigClientType.swift

```diff
@@ -0,0 +1,25 @@
+import FirebaseRemoteConfig
+import Foundation
+
+public protocol RemoteConfigClientType: AnyObject {
```

Comment:
This was your protocol - the reason it wasn't mapping to the `RemoteConfig.remoteConfig()` object was because that was ObjC. And - I had to look this up - there's quite a bit of annoying bridging that needs to be done when conforming ObjC to Swift code even if its' a 1-1 protocol. Which leads me to...